docs/sdk/src/reference_docs/chain_spec_runtime/tests/chain_spec_builder_tests.rs

```diff
@@ -0,0 +1,128 @@
+use serde_json::{json, Value};
+use std::{process::Command, str};
+
+const WASM_FILE_PATH: &str =
+	"../../../../../target/release/wbuild/chain-spec-guide-runtime/chain_spec_guide_runtime.wasm";
+
+const CHAIN_SPEC_BUILDER_PATH: &str = "../../../../../target/release/chain-spec-builder";
+
+fn get_chain_spec_builder_path() -> &'static str {
+	// dev-dependencies do not build binary. So let's do the naive work-around here:
```

Comment:
I have been brainstorming with how e can format this nicely into the docs with @rzadp, and the outcome is https://github.com/sam0x17/docify/issues/26